### PR TITLE
CASSANDRA-19442 Update access of ClearSnapshotStrategy

### PR DESCRIPTION
**Pull request by sarankk**

### CODE REVIEW

File: cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/ClientConfig.java [hunk 1]
```diff
@@ -39,9 +39,9 @@
 
 import static org.apache.cassandra.spark.data.CassandraDataLayer.aliasLastModifiedTimestamp;
 
-public final class ClientConfig
+public class ClientConfig
 {
-    private static final Logger LOGGER = LoggerFactory.getLogger(ClientConfig.class);
+    protected static final Logger LOGGER = LoggerFactory.getLogger(ClientConfig.class);
```
Comment (frankgh) [inline cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/ClientConfig.java]:
```suggestion
    protected final Logger logger = LoggerFactory.getLogger(this.getClass());
```

**Review by frankgh — COMMENTED** [review]
Looks good, I've left a minor comment

**Review by frankgh — APPROVED** [review]
+1 Thanks for the patch

**Comment by frankgh** [comment]
Closed via https://github.com/apache/cassandra-analytics/commit/cf6de14d5b96ea173d6a1b2dad9bb64d563df06c

**Comment by milkg0d** [comment]
```
[pom.xml.xlsx](https://github.com/apache/cassandra-analytics/files/15100379/pom.xml.xlsx)
```

**Pull request closed without merging**